doc/changes.rst

```diff
@@ -5,7 +5,8 @@ desitarget Change Log
 0.37.1 (unreleased)
 -------------------
 
-* No changes yet.
+* Fixes a typo in the priority of MWS_WD_SV targets [`PR #600`_].
```

Comment:
You need a blank line after this line (bullet lists have to end with a blank line in reStructuredText).